moj-danger.gemspec

```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = 'moj-danger'
+  s.version = '0.1.0'
+  s.authors = ['Ministry Of Justice']
+  s.email = ['tools+danger@digital.justice.gov.uk']
+  s.license = 'MIT'
+  s.date = '2017-05-12'
+
+  s.required_ruby_version = '>= 2.3.0'
+
+  s.summary = 'Add Danger and linter gems'
+  s.homepage = 'https://github.com/ministryofjustice/danger'
+
+  s.files = %w(README.md LICENSE)
```

Comment:
Adding the `Dangerfile` to the gem wouldn't do anything useful, IMO. It would end up in whatever location the gem got installed. You'd have to add a  command script to the gem to create/copy the `Dangerfile` to the local project (i.e. `moj-danger init`)

Since we give instructions in the README to create the appropriate Dangerfile, I didn't think it was useful to have it here.